### DISCUSSION TITLE
Why is the emissive textures so dim?

### DISCUSSION THREAD

**Discussion by OHAce** [MoreMcmeta/core#82] [Help Wanted]
My emissive texture packs are extremely dim on Mcmeta compared to optifine and I'm trying to figure out why.
Emissive
![2024-05-31_16 14 00](https://github.com/MoreMcmeta/core/assets/111223268/590dee0d-138d-4c34-9060-1ca85591999e)
Non-emissive
![2024-05-31_16 14 44](https://github.com/MoreMcmeta/core/assets/111223268/d7462eda-62c1-43c4-800e-7fc185bccc6b)

I am using Forge 1.20.1. I have the standard performance mods such as rubidium, ferrite core and modernfix. I did look at the other thread where someone didn't have them show up due to modernfix dynamic resources, but my issue is that the emissive textures are extremely dim, not that they don't render.

**Comment by OHAce** [comment]
![2024-05-31_18 31 48](https://github.com/MoreMcmeta/core/assets/111223268/66327e5a-4c81-48c4-a131-96e21ce37f30)

As it would turn out, the issue was Rubidium.

**Comment by soir20** [comment]
Yes, this is a known issue with Rubidium and would need to be fixed on their side: https://github.com/MoreMcmeta/emissive-plugin/issues/2

Someone says that it was already fixed--perhaps just a Rubidium version update is needed?
